### PR TITLE
feat: Add goal tracking to delete all menses tags, closes #304

### LINKED ISSUE #304: Add goal tracking to delete all menses tags feature
See instruction in #302

### PR DESCRIPTION
**Pull request by olavea**

### CODE REVIEW

**Comment by olavea** [comment]
Take a look @raae 👍

File: src/constants.js [hunk 1]
```diff
@@ -22,6 +22,10 @@ export const FATHOM_CHANGELOG = process.env.GATSBY_FATHOM_CHANGELOG
 export const FATHOM_MADE_BY = process.env.GATSBY_FATHOM_MADE_BY
 export const FATHOM_SUPPORT = process.env.GATSBY_FATHOM_SUPPORT
 export const FATHOM_EXPORT = process.env.GATSBY_FATHOM_EXPORT
+export const FATHOM_DELETE_ALL_MENSES_TAGS = process.env.GATSBY_FATHOM_DELETE_ALL_MENSES_TAGS
+export const FATHOM_DELETE_TAGS_CONFIRMED = process.env.GATSBY_FATHOM_DELETE_TAGS_CONFIRMED
```
Comment (raae) [inline src/constants.js:26]: **suggestion:** No need for confirmed on this as there is no confirmation process as it is for delete entries.

File: src/features/profile/DangerCard.js [hunk 2]
```diff
@@ -15,6 +15,9 @@ import { CardContentSection } from "../../components"
 import { selectAllEntries, deleteAllEntries } from "../entries"
 import { useSettings } from "../settings"
 
+import { FATHOM_DELETE_ALL_MENSES_TAGS, FATHOM_DELETE_TAGS_CONFIRMED } from "../../constants";
```
Comment (raae) [inline src/features/profile/DangerCard.js:18]: **nitpick (non-blocking):** I like to have the imports with more `..` above the once with less.

**Review by raae — CHANGES_REQUESTED** [review]

File: src/constants.js (hunk 1 above)
Comment (olavea) [reply src/constants.js]: suggestion: No need for confirmed #304
b0a52c8

**Comment by olavea** [comment]
Take a look Queen @raae 👑 ?

File: src/constants.js [hunk 3]
```diff
@@ -22,6 +22,10 @@ export const FATHOM_CHANGELOG = process.env.GATSBY_FATHOM_CHANGELOG
 export const FATHOM_MADE_BY = process.env.GATSBY_FATHOM_MADE_BY
 export const FATHOM_SUPPORT = process.env.GATSBY_FATHOM_SUPPORT
 export const FATHOM_EXPORT = process.env.GATSBY_FATHOM_EXPORT
+export const FATHOM_DELETE_ALL_MENSES_TAGS = process.env.GATSBY_FATHOM_DELETE_ALL_MENSES_TAGS
+
```
Comment (raae) [inline src/constants.js]: **nitpick (non-blocking):** Remove empty lines

File: src/features/profile/DangerCard.js [hunk 4]
```diff
@@ -10,15 +10,14 @@ import {
 } from "@material-ui/core"
 import { ErrorOutline as DangerIcon } from "@material-ui/icons"
 
+import { FATHOM_DELETE_ALL_ENTRIES, FATHOM_DELETE_ALL_ENTRIES_CONFIRMED, FATHOM_DELETE_ALL_MENSES_TAGS } from "../../constants";
```
Comment (raae) [inline src/features/profile/DangerCard.js:13]: **nitpick (non-blocking):** Pull down the changes locally and give it a save so its formatted properly.

**Review by raae — APPROVED** [review]

File: src/features/profile/DangerCard.js (hunk 4 above)
Comment (olavea) [reply src/features/profile/DangerCard.js:13]: I don't understand, so I will learn this some other time

**Comment by olavea** [comment]
365DaysOfPOWprogramming-day-157_feat- Add  goal tracking to delete all menses tags, closes  #304 #312 -375

**Pull request merged**